pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -158,12 +158,20 @@ def cmd(self):
 
         if self.disable_capture:
             cmd.append("-s")
-
         if self.xdist_ip_addresses:
             cmd.append('--dist=loadscope')
-            for ip in self.xdist_ip_addresses.split(' '):
-                xdist_string = '--tx ssh=ubuntu@{}//python="source /edx/app/edxapp/edxapp_env; ' \
-                               'python"//chdir="/edx/app/edxapp/edx-platform"'.format(ip)
+            if self.processes < 0:
```

Comment:
Shouldn't this be `<= 0` like the one below?  Otherwise it seems like we ask for exactly 0 workers by default in `xdist_string` below.

Reply:
yep. good eyes 👀